docs/scripting-doc/index.d.ts

```diff
@@ -551,7 +551,7 @@ declare namespace Qt {
      * The amount that each tick left or right on the slider will
      * change the value.
```

Comment:
Obviously a copy/paste issue from the `QSlider`. This comment needs to be adjusted as well, could be just the short form "This property holds the step value."

Reply:
Thanks - I didn't notice that!  Updated the comment per your suggestion.